ui/imports/shared/views/chat/LinksMessageView.qml

```diff
@@ -212,17 +212,18 @@ Column {
         id: unfurledImageComponent
 
         MessageBorder {
-            width: linkImage.width
-            height: linkImage.height
+            width: linkImage1.width
+            height: linkImage1.height
             isCurrentUser: root.isCurrentUser
             StatusChatImageLoader {
-                id: linkImage
+                id: linkImage1
```

Comment:
looks like some anti-pattern `x1, x2, x3, x4`. Could you pls add more informative `id`?

Reply:
@borismelnik we had the same id in the same component before, so adding 1 and 2 to differ them is an improvement. :) Ok, will try to come up with some new, more creative names. ;)